src/components/RequireData.tsx

```diff
@@ -2,10 +2,12 @@ import React from 'react';
 import { connect } from 'react-redux';
```

Comment:
This file should probably be removed now that we have `bootstrap` action creator